libs/components/theme/src/lib/styles/_switch.scss

```diff
@@ -72,7 +72,11 @@
     }
 
     &:focus + .sky-switch-control {
-      @include mixins.sky-focus-outline;
+      //default
+      outline: thin dotted;
```

Comment:
This was the last place that the `sky-focus-outline` mixin was used so I converted these to use the mixin's contents.